test/framework/src/main/java/org/opensearch/test/OpenSearchIntegTestCase.java

```diff
@@ -1731,7 +1730,17 @@ protected Set<List<String>> indexRandomForMultipleSlices(String... indices) thro
                 refresh(index);
             }
         }
-        return bogusIds;
+        if (!bogusIds.isEmpty()) {
```

Comment:
I don't think you need the if statement since the for loop is a no-op for an empty set.

Reply:
makes sense, will update it